go.mod

```diff
@@ -86,6 +86,7 @@ require (
 	github.com/go-openapi/jsonreference v0.19.5 // indirect
 	github.com/go-openapi/swag v0.19.14 // indirect
 	github.com/gobuffalo/flect v0.2.4 // indirect
+	github.com/gofrs/uuid v4.2.0+incompatible // indirect
```

Comment:
Is this change intentional?

Reply:
It was pulled in by `make modules` after I added

```go
import "github.com/Azure/azure-sdk-for-go/services/msi/mgmt/2018-11-30/msi"
```